packages/Panel/README.md

```diff
@@ -49,28 +97,32 @@ npm install @paprika/panel
 You can use any of the following components to compose the Panel:
 
 ```js
+<Panel.Content />
 <Panel.Header />
 <Panel.Footer />
 <Panel.Overlay />
 <Panel.FocusLock />
 <Panel.Trigger />
 ```
 
-## Basic examples.
+## Basic examples
 
 ```js
-<Panel isOpen={isOpen} onClose={() => {}}>
-  [...your_content...]
-</Panel>
+import Panel from "@paprika/panel";
+
+const [isOpen, setIsOpen] = React.useState(true);
+
+<Panel isOpen={isOpen} onClose={() => {}} slideFrom={Panel.types.slideFrom.LEFT}>
```

Comment:
Since you're demonstrating a working controlled example, you should probably have an actual close function for `onClose`. 

Can probably save any non-required props, like `slideFrom` and `kind` for the docs page and keep this as simple as possible.